buildenv/Build_Instructions_V8.md

```diff
@@ -314,6 +314,12 @@ OpenJ9 uses the mingw/GCC compiler during the build process. In the `Archive` ca
 - [Freemarker V2.3.8](https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download)
 - [Freetype2 V2.3 or newer](https://www.freetype.org/)
 
+Note:
+For building a 32-bit version, please install 32-bit Cygwin and all required packages must be installed via Cygwin as follows:
```

Comment:
Is this really necessary? If cygwin is installed as described above, is there anything missing which is required for 32-bit? These instructions imply that mingw64-i686-gcc-g++ is missing, but isn't this part of the `Devel` category? If not, what category is it in?

Reply:
We'd like to be able to install a single cygwin installation which is suitable for building for 32-bit and 64-bit.

Reply:
32 bit Cygwin might be unnecessary but mingw64-i686-gcc-g++ might not be installed automatically by default.
I think it is reasonable for users to install mingw64-i686-gcc-g++ if compilation failure prompts them to do so

Reply:
The instructions say `Install all packages in the `Devel` category, which includes `mingw`, a minimalist subset of GNU tools for Microsoft Windows.`. I'm hoping the `Devel` category includes mingw64-i686-gcc-g++, we'll have to check. Otherwise we need specific instructions for how to find it in the cygwin installer.

Reply:
Already double-checked that mingw64-i686-gcc-g++ is included in Devel category.

Reply:
In that case there shouldn't need to be any special note about cygwin for building 32-bit.

Reply:
Agreed.